airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -34,6 +34,7 @@
 class SalesforceStream(HttpStream, ABC):
     page_size = 2000
     transformer = TypeTransformer(TransformConfig.DefaultSchemaNormalization)
+    encoding = "ISO-8859-1"
```

Comment:
There is any link to Salesforce docs explaining or saying about this encoding?

Reply:
https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/implementation_considerations.htm